monitoring/harmony-monitor/blockchain-watchdog/reporting-server.go

```diff
@@ -81,9 +81,11 @@ func init() {
 	for i := 0; i < h.NumField(); i++ {
 		headerInformationCSVHeader = append(headerInformationCSVHeader, h.Field(i).Name)
 	}
+	headerInformationCSVHeader = append([]string{"IP"}, headerInformationCSVHeader...)
```

Comment:
Don't do these happens with `{"IP"}`, create the original slice with the IP entry already in it, on https://github.com/harmony-one/harmony-ops/blob/a26143c2d465a7f4dcb2cf0e6bb2a8bb551c385e/monitoring/harmony-monitor/blockchain-watchdog/reporting-server.go#L56

Reply:
Smarter. Will change.